en/drafts/originals/transcribe-handwritten-text-with-python-and-azure.md

```diff
@@ -561,10 +534,6 @@ White, William. 1917. William Andrew White fonds, R15535-0-8-E, "1917 Diary", It
 
 Cognitive-services-quickstart-code, June 22, 2021, https://docs.microsoft.com/en-us/azure/cognitive-services/computer-vision/quickstarts-sdk/python-sdk. Accessed 25 December, 2021.
 
-## Footnotes
+## Endnotes
```

Comment:
I found most of the endnotes functioned more effectively as embedded links within the text.